mod.ts

```diff
@@ -500,3 +504,36 @@ export interface CustomSMSGatewayPayload {
   to: string;
   body: string;
 }
+
+// Account migration
+export interface AccountMigrationRequest {
+  migration_token: string;
+}
+
+export interface AccountMigrationResponse {
+  identities?: IdentityMigrateSpec[];
+  authenticators?: AuthenticatorMigrateSpec[];
+}
+
+export interface IdentityMigrateSpec {
```

Comment:
We can have even stricter type definition here by using a type union. See https://www.typescriptlang.org/play?#code/C4TwDgpgBAMg9gcwJYDsCSARAKuaBeKAIggFsBDJAG0KgB8iwALOFCG+wgVwGcIAnFGRJsA3ACgxoSFDQATCCmBJQAWSQI+ZYBADKkAMZQCchUtDxk6DGo1bdB8WNTa+AMzL7oJxcpAXUmDaa2noQhgDeYlBQUhAAXESUiKgA+kiyhOLRSZZpsgmR0dEA1hAgCdzAfKgIWUWxCf5WOJB10QBuZJSc8VCV1Si1UVAAvmJjkrhQAIKcwIymSPpacHxBdqGGBLPzi8vAqwDyhwBCh1gACgCi5FTrIQZ0M3MLPvtHp+cXOio-6sH2MKOZz8dyeZ67N4rPjHM6XG4USj3QERYYNIhwOAAIxScGAYBSpERmWGmKxeLABWG0SJVAqVRqdTGExBbg80B2ryU7xhn0uPz+tgeYSghRiuAShDJuPxKW4JG4JOiZIpVKKUCYLF6-UZw2ZQA